django-reference-app/django-reference-app/static/service_settings.py

```diff
@@ -0,0 +1,118 @@
+##view - using template as a view opposed to 
```

Comment:
Has this file been generated by the `django-govuk-forms` package? I'm not really clear what it is or what it's doing, but if it's boilerplate from that package we should probably go with it.

Either way though, is it supposed to be in the `static` directory? That's normally used exclusively for JS, CSS and images.